Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -136,18 +138,36 @@ class SnackBar: UIView {
         backgroundView.snp.makeConstraints { make in
             make.bottom.left.right.equalTo(self)
             // Offset it by the width of the top border line so we can see the line from the super view
-            make.top.equalTo(self).offset(1)
+            make.top.equalTo(self)//.offset(1)
         }
 
         titleView.snp.makeConstraints { make in
             make.top.equalTo(self).offset(UIConstants.DefaultPadding)
+            make.height.equalTo(UIConstants.SnackbarButtonHeight - 2*UIConstants.DefaultPadding)
             make.centerX.equalTo(self).priority(500)
-            make.width.lessThanOrEqualTo(self).inset(UIConstants.DefaultPadding * 2).priority(1000)
+            make.width.lessThanOrEqualTo(self).inset(UIConstants.DefaultPadding * 6).priority(1000)
         }
 
         backgroundColor = UIColor.clear
+        self.clipsToBounds = true //overridden by masksToBounds = false
         self.layer.borderWidth = SnackBarUX.BorderWidth
-        self.layer.borderColor = UIConstants.BorderColor.cgColor
+        self.layer.borderColor = UIConstants.SeparatorColor.cgColor
+        self.layer.cornerRadius = 8
+        if #available(iOS 11.0, *) {
+            self.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner]
+        } else {
+            // Fallback on earlier versions
```

Comment:
How is this handled in older iOS versions? If we dont need to do anything for those we can remove the else statement

Reply:
Without this if statement, all 4 corners are rounded. I don't think there's a straightforward way to round specified corners in earlier iOS versions. I took out the else statement for now.